src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/StringBuilderAppendShouldNotTakeSubstring.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System.Collections.Immutable;
+using Analyzer.Utilities;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
```

Comment:
This is a layering violation. `Core` layer is meant for language-agnostic code only.

Reply:
oh, that explains why my VB tests didn't work... 
I'll give it a try...